bundler/spec/dependabot/bundler/update_checker_spec.rb

```diff
@@ -1383,10 +1383,6 @@
       let(:dependency_name) { "guard-bundler" }
       let(:current_version) { "2.2.1" }
 
-      context "when using bundler v1", :bundler_v1_only do
-        it { is_expected.to eq(Gem::Version.new("2.2.1")) }
-      end
-
       context "when using bundler v2", :bundler_v2_only do
```

Comment:
What is this `:bundler_v2_only`? Is this test still relevant or was it only needed when we had both v1 and v2?

Reply:
Yes, We don't need this check. I removed all checks for bundler_v2_only. Specs are going to run without this check.  I think this was used to loop tests for both bundlers. But we don't need to loop anymore.